grpc-examples/src/main/java/io/stargate/grpcexample/GrpcClientExecuteQuery.java

```diff
@@ -221,6 +223,35 @@ public void executeSingleQuery() {
         "v = " + rs.getRows(0).getValues(1).getInt()); // it will return value for v = 1
   }
 
+  public void executeSingleQueryParameterized() {
+    // insert
+    QueryOuterClass.Response response =
+        blockingStub.executeQuery(
+            QueryOuterClass.Query.newBuilder()
+                .setCql("INSERT INTO ks.test (k, v) VALUES (?, ?)")
+                .setValues(
+                    QueryOuterClass.Values.newBuilder()
+                        .addValues(Values.of("b"))
+                        .addValues(Values.of(2))
+                        .build())
+                .build());
+    System.out.println(response);
+    // retrieve
+    response =
+        blockingStub.executeQuery(
+            QueryOuterClass.Query.newBuilder()
+                .setCql("SELECT k, v FROM ks.test WHERE k = ?")
+                .setValues(QueryOuterClass.Values.newBuilder().addValues(Values.of("b")).build())
+                .build());
+
+    QueryOuterClass.ResultSet rs = response.getResultSet();
+
+    System.out.println(
```

Comment:
I assume these were accidentally left during testing? Should be removed.

Reply:
Remember that this is example code that the user is meant to run (likely in their IDE) after setting the Astra token.  It's supposed to print stuff out.

Reply:
Ah. Ok makes sense then. Forgot the context. Thanks!